libs/service-portal/finance/src/screens/FinanceTransactionPeriods/FinanceTransactionPeriods.graphql

```diff
@@ -0,0 +1,71 @@
+query getAssessmentYears {
+  getAssessmentYears {
+    year
+  }
+}
+
+query GetChargeTypesByYear($input: GetChargeTypesByYearInput!) {
+  getChargeTypesByYear(input: $input) {
+    chargeType {
+      iD
+      name
+    }
+  }
+}
+
+query GetChargeTypesDetailsByYear($input: GetChargeTypesDetailsByYearInput!) {
+  getChargeTypesDetailsByYear(input: $input) {
+    chargeType {
+      iD
+      name
+      chargeItemSubjects
+      chargeItemSubjectDescription
+      lastMovementDate
+    }
+  }
+}
+
+query GetChargeItemSubjectsByYear($input: GetChargeItemSubjectsByYearInput!) {
+  getChargeItemSubjectsByYear(input: $input) {
+    chargeItemSubjects {
+      chargeItemSubject
+      lastMoveDate
+      totalAmount
+      periods {
+        period
+        description
+        lastMoveDate
```

Comment:
I feel in my heart of hearts that this could be optimized somewhat. So many queries for similar looking data